src/main/webapp/pages/common/error/defaultNotFoundPage.jsp

```diff
@@ -107,7 +107,7 @@ $(document).ready(function () {
     var referrerPath = document.referrer;
     var referrer = '<spring:message code="errorpage.previous" text="errorpage.previous"/><br /><span class="troubleshooting">' + referrerPath + '"</span>"';
     if (referrerPath != '') {
-            $("#previous-path").html(referrer);
+            $("#previous-path").html(Encode.forJavaScript(referrer));
```

Comment:
I think we should use decodeURIComponent here. This isn't being output in a java scriptlet so we wouldn't have access to Encode.forJavaScript